packages/firestore/src/api/blob.ts

```diff
@@ -51,16 +52,7 @@ function assertBase64Available(): void {
  * Note that while you can't hide the constructor in JavaScript code, we are
  * using the hack above to make sure no-one outside this module can call it.
  */
-export class Blob {
-  // Prefix with underscore to signal that we consider this not part of the
-  // public API and to prevent it from showing up for autocompletion.
-  _byteString: ByteString;
-
-  constructor(byteString: ByteString) {
-    assertBase64Available();
-    this._byteString = byteString;
-  }
-
+export class Blob extends Bytes {
```

Comment:
optional: It might be helpful to add some context in comments about why the main SDK's `Blob` is extending the `Bytes` class in the Lite SDK. It's clear to me what's happening when I look at the PR description, but without it, I would be confused.

Reply:
Done (but missed this before merging - coming in the next PR).